streamz/sources.py

```diff
@@ -453,12 +453,13 @@ def _close_consumer(self):
 class FromKafkaBatched(Stream):
     """Base class for both local and cluster-based batched kafka processing"""
     def __init__(self, topic, consumer_params, poll_interval='1s',
-                 npartitions=1, **kwargs):
+                 npartitions=1, keys=False, **kwargs):
         self.consumer_params = consumer_params
         self.topic = topic
         self.npartitions = npartitions
         self.positions = [0] * npartitions
         self.poll_interval = convert_interval(poll_interval)
+        self.keys = keys
```

Comment:
This is already in master?

Reply:
I removed this and it will be merged in when pulled to master.

Reply:
What about changes like this one that would be needed: [checkpointing + keys](https://github.com/chinmaychandak/streamz/blob/a951ffa91e626a8ce2a00fee16ac7309a5a693ce/streamz/sources.py#L473).